src/CMakeLists.txt

```diff
@@ -1264,9 +1119,9 @@ if (BUILD_STATIC_LIBS)
 endif ()
 
 if (BUILD_SHARED_LIBS)
-  set (shared_gen_SRCS ${HDF5_GENERATED_SOURCE_DIR}/shared/H5Tinit.c ${HDF5_SRC_BINARY_DIR}/shared/H5lib_settings.c)
+  set (shared_gen_SRCS ${HDF5_SRC_BINARY_DIR}/H5build_settings.c)
```

Comment:
This shared_gen_SRCS variable could also be deleted and just use ${HDF5_SRC_BINARY_DIR}/H5build_settings.c directly.